docs/Salesforce-streamingsource.md

```diff
@@ -69,6 +69,46 @@ for the evaluated records only if they match the criteria specified in the WHERE
 Salesforce and generate SOQL query (`select <FIELD_1, FIELD_2, ..., FIELD_N> from ${sObjectName}`).
 Ignored if SOQL query is provided.
 
+There are also **sObjects** that are not supported in the Bulk API of Salesforce.
+When a job is created using an object that is not supported in the Bulk API, "_Entity is not supported by the Bulk API_" is thrown.
```

Comment:
when would plugin use build api? How would plugin user know it?

Reply:
are we using bulk api in streaming source?

Reply:
Thanks for catching this. Streaming source does not use Bulk API. I removed it from the documentation of Salesfoce-streamingsource in the last update.